documentation/IDTA-01001/modules/ROOT/pages/includes/idshortpath.adoc

```diff
@@ -12,9 +12,12 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, d
 
 === Format "Path" (IdShortPath Serialization) in JSON
 
-To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an JSON array. 
-The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is present. In the first case, the submodel element's idShort 
-is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[<Index>]" are appended after the idShort of the containing SubmodelElementList.
+To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms 
+of an idShortPath notation to be returned in an JSON array. 
+For SubmodelElementLists the elements can be identified via its index or via its idShort.
+For identification via idShort the submodel element's idShort 
+is separated by "." (dot) from top level down to child level. 
```

Comment:
there needs to be a final decision on handling with "." notation or via brackets